.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -39,17 +41,33 @@ export class ProspectEvents extends Resource {
     super(scope, name);
 
     // pre-existing queues (prod and dev) created by prospect-api
-    this.sqs = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-queue`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`
-    })
+    this.sqs = new sqs.DataAwsSqsQueue(
+      this,
+      `prospect-${config.environment}-queue`,
+      {
+        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`,
+      }
+    );
+
+    this.sqsDlq = new sqs.DataAwsSqsQueue(
+      this,
+      `prospect-${config.environment}-dlq`,
+      {
+        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`,
+      }
+    );
 
-    this.sqsDlq = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-dlq`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`
+    this.snsTopic = new sns.SnsTopic(this, 'prospect-event-topic', {
+      name: `${config.prefix}-ProspectEventTopic`,
```

Comment:
is this sns topic specific to dismissing prospects? or a general sns topic to be used for many/all prospect events?

Reply:
this SNS topic is target for this event-rule so we can connect more than 5 targets to this rule if needed (event-bridge restricts 5 targets per rule, so we are adopting this pattern)

right now, this has source as `prospect-events` and detail-type as `dismiss-prospects`
but in future, say we have another event (e.g generate-prospect), we can add them to detail-type. 

we already have a prospect-events rule but we didn't add this target to the existing rule coz it already forwards to another translation SQS/dev event bridge. didn't want to have any underlying implications on that workflow. this wa